activerecord/test/models/seminar.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+##
+# Represents a seminar.
```

Comment:
Comments are good 👍 but I don't know if we need these. None of the other test models seem to be commented.